pkg/appliance/functions.go

```diff
@@ -910,12 +910,12 @@ func downloadDockerImageBundle(args imageBundleArgs) {
 		}
 		res, err := args.client.Get(fmt.Sprintf("https://public.ecr.aws/token/?scope=repository:appgate-sdp/%s:pull&service=public.ecr.aws", args.image))
 		if err != nil {
-			args.fileEntryChan <- fileEntry{err: err}
+			args.fileEntryChan <- fileEntry{err: api.HTTPErrorResponse(res, fmt.Errorf("failed to fetch public ECR token: %w", err))}
```

Comment:
`api.HTTPErrorResponse` is design with the intent to handle error response from the appgate controller, it does not really serve any purpose to use it on other domains.